pandas/core/config_init.py

```diff
@@ -297,6 +311,10 @@ def mpl_style_cb(key):
                        pc_east_asian_width_doc, validator=is_bool)
     cf.register_option('unicode.ambiguous_as_wide', False,
                        pc_east_asian_width_doc, validator=is_bool)
+    cf.register_option('latex.escape',False, pc_latex_escape,
+                        validator=is_bool)
+    cf.register_option('latex.longtable',False,pc_latex_longtable,
+                        validator=is_bool)
```

Comment:
``.escape`` is True by default in ``to_latex``. So the default should be ``True`` here as well.